### PR TITLE
Added MATLAB support

### PR DESCRIPTION
**Pull request by bremme**
Added support for MATLAB. Tested on Linux using MATLAB 2016a.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,7 @@ Currently supported grammars are:
 | LiveScript                           | Yes        | Yes             | |
 | Lua                                  | Yes        | Yes             | |
 | Makefile                             | Yes        | Yes             | |
+| [MATLAB](http://mathworks.com/products/matlab) | Yes        | Yes             | Requires the language-matlab package in Atom https://atom.io/packages/language-matlab |
```
Comment (nixel2007) [inline README.md]: Looks like there is nothing special in language-matlab package. For grammar support we have a `NOTE` section at the bottom of the table.

Did i miss something and this package is necessary not only fo grammar?

**Comment by nixel2007** [comment]
@bremme hi and thanks for the PR!

**Comment by bremme** [comment]
You are right, the language-matlab package is only required for grammar support. Shall I make a new commit and remove it?

**Comment by nixel2007** [comment]
Yes, please.

**Comment by nixel2007** [comment]
:ship:

**Pull request merged**